src/aliases.rs

```diff
@@ -67,3 +68,41 @@ pub type TimerInstantU32<const FREQ_HZ: u32> = Instant<u32, 1, FREQ_HZ>;
 
 /// Alias for instants that come from timers with a specific frequency (`u64` backing storage)
 pub type TimerInstantU64<const FREQ_HZ: u32> = Instant<u64, 1, FREQ_HZ>;
+
+// -------------------------------
+
+/// Alias for hertz rate
+pub type Hertz<T> = Rate<T, 1, 1>;
+
+/// Alias for hertz rate (`u32` backing storage)
+pub type HertzU32 = Rate<u32, 1, 1>;
+
+/// Alias for hertz rate (`u64` backing storage)
+pub type HertzU64 = Rate<u64, 1, 1>;
+
+/// Alias for kilohertz rate
+pub type Kilohertz<T> = Rate<T, 60, 1>;
+
+/// Alias for kilohertz rate (`u32` backing storage)
+pub type KilohertzU32 = Rate<u32, 60, 1>;
+
+/// Alias for kilohertz rate (`u64` backing storage)
+pub type KilohertzU64 = Rate<u64, 60, 1>;
+
+/// Alias for megahertz rate
+pub type Megahertz<T> = Rate<T, 3_600, 1>;
+
+/// Alias for megahertz rate (`u32` backing storage)
+pub type MegahertzU32 = Rate<u32, 3_600, 1>;
+
+/// Alias for megahertz rate (`u64` backing storage)
+pub type MegahertzU64 = Rate<u64, 3_600, 1>;
```

Comment:
I think you copied and pasted wrong here

Reply:
Thanks, fixed